docs/data.md

```diff
@@ -151,3 +151,128 @@ i.e., push the computation region settings high in the code structure.
 However, for running a job, the computational region should be somewhere
 otherwise it will rely on whatever is the (last set) computational region in the
 current mapset.
+
+## Using local scratch space for temporary data
+
+### Running a single job
+If you are processing a large number of temporary files with GRASS GIS, it can be useful to write the   
+temporary files to local scratch space on the compute node (can decrease run time). In your  
+[job submission script](https://github.com/ncsu-geoforall-lab/grass-gis-on-hpc-henry2/blob/main/docs/jobs.md#running-a-single-job-calling-a-grass-module) you will need to add the following code, e.g.:  
+
+```tcsh
+#!/bin/tcsh
+#BSUB -n 1
+#BSUB -W 48:00
+#BSUB -R span[hosts=1]
+#BSUB -oo comp1_out
+#BSUB -eo comp1_err
+#BSUB -J comp1
+
+#########   LOAD MODULES   #########
```

Comment:
I'm generally against these many-hashes-all-caps comments. I think a full sentence ending with a period is generally more useful. Python's PEP8/flake8 would reject multiple `#` characters at the beginning of the line.